dev/com.ibm.ws.jsp/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -20,7 +20,13 @@
             id="disableJspRuntimeCompilation" required="false" type="Boolean" default="false"/>
         <AD description="%extendedDocumentRoot" name="%extendedDocumentRoot.name"
             id="extendedDocumentRoot" required="false" type="String" default="" /> 
-        <AD description="%jdkSourceLevel" name="%jdkSourceLevel.name"
+        <AD description="%javaSourceLevel" name="%javaSourceLevel.name"
+            id="javaSourceLevel" required="false" type="String">
+            <Option label="8" value="8"/>
+            <Option label="11" value="11"/>
+            <Option label="17" value="17"/>
+        </AD> 
+        <AD name="internal" description="internal use only"
```

Comment:
Should we add a comment as to why we marked this as internal so it is clear in the future when someone looks at this?

Reply:
We should also update the copyright date range header to include 2023

Reply:
Can we remove the contributors section of the copyright header here as well?